services/src/integ-test/resources/application-integ.yaml

```diff
@@ -9,9 +9,9 @@ spring:
     servicebus:
       internal:
         targetAppQueue:
-          MCOL: ${CIVIL_SDT_QUEUE:Test1Queue}
-          MCOLS: ${CIVIL_SDT_QUEUE:Test1Queue}
```

Comment:
we can keep this and change defaults

Reply:
Added back the CIVIL_SDT_QUEUE environment variable part to these entries.